snips_nlu/cli/dataset/entities.py

```diff
@@ -56,16 +57,25 @@ def from_file(cls, filepath):
             if six.PY2:
                 it = list(utf_8_encoder(it))
             reader = csv.reader(list(it))
+            autoextent = True
             for row in reader:
                 if six.PY2:
                     row = [cell.decode("utf-8") for cell in row]
                 value = row[0]
+                m = re.match(
+                    r'^#\sautomatically_extensible=(true|false)\s*$',
+                    row[0], re.IGNORECASE)
+                if m:
+                    autoextent = not m.group(1).lower() == 'false'
+                    continue
+                if value.startswith('#'):
+                    continue
                 if len(row) > 1:
                     synonyms = row[1:]
                 else:
                     synonyms = []
                 utterances.append(EntityUtterance(value, synonyms))
-        return cls(entity_name, utterances, automatically_extensible=True,
+        return cls(entity_name, utterances, autoextent,
```

Comment:
It's probably a good idea to leave the parameter name here to avoid silent errors in case new boolean parameters are introduced.